build.gradle

```diff
@@ -118,6 +118,7 @@ def createSpotlessTarget = { pattern ->
         'charts', // Helm charts often have injected template strings that will fail general linting. Helm linting is done separately.
         'resources/seed/*_specs.yaml', // Do not remove - this is necessary to prevent diffs in our github workflows, as the file diff check runs between the Format step and the Build step, the latter of which generates the file.
         'airbyte-integrations/connectors/source-amplitude/unit_tests/api_data/zipped.json', // Zipped file presents as non-UTF-8 making spotless sad
+        'airbyte-webapp/src/test-utils/mock-data/mockDestinationDefinition.json',
```

Comment:
Note to self: remove this change before merging.

I just added this to get the build to pass since something weird is up with the auto-formatting on this file (seems like prettier pre-commit hook removes changes that spotless wants to have) -- I'll ping KC about this since he added this file today